lib/inets/doc/src/httpc.xml

```diff
@@ -143,593 +72,809 @@
       The configuration options are as follows:</p>
     <taglist>
       <tag>{profile, profile()}</tag>
-      <item><p>Name of the profile, see
+      <item>Name of the profile, see
 	<seeerl marker="#DATA_TYPES">DATA TYPES</seeerl>.
-      This option is mandatory.</p></item>
+      This option is mandatory.</item>
       <tag>{data_dir, path()}</tag>
-      <item><p>Directory where the profile
+      <item>Directory where the profile
 	can save persistent data. If omitted, all cookies are treated
-	as session cookies.</p></item>
+	as session cookies.</item>
     </taglist>
 
-    <p>The client can be stopped using <c>inets:stop(httpc, Pid)</c> or
-    <c>inets:stop(httpc, Profile)</c>.</p>
+    <p>The client can be stopped using <seemfa marker="inets#stop/2"><c>inets:stop(httpc, Pid)</c></seemfa> or
+    <seemfa marker="inets#stop/2"><c>inets:stop(httpc, Profile)</c></seemfa>.</p>
+    <marker id="DATA_TYPES"></marker>
+    <warning><p>Please note that <c>httpc</c> normalizes input URIs before internal
+    processing and special care shall be taken when the URI has percent ("%")
+    characters. A percent serves as the indicator for percent-encoded octets and
+    it must be percent-encoded as "%25" for that octet to be used as data within
+    the URI.</p>
+    <p>For example, in order to send an <c>HTTP GET</c> request with the
+    URI <c>http://localhost/foo%25bar</c>, the percent character must be
+    percent-encoded when creating the request:
+    <c>httpc:request("http://localhost/foo%2525bar").</c> </p></warning>
   </section>
-  
+
+
+  <datatypes>
+    <datatype>
+      <name name="request_id" />
+    </datatype>
+    <datatype>
+      <name name="profile" />
+    </datatype>
+    <datatype>
+      <name name="socket_opt"/>
+      <desc>
+        <p>See the options used by
+        <seeerl marker="kernel:gen_tcp">gen_tcp(3)</seeerl> and
+        <seeerl marker="ssl:ssl">ssl(3)</seeerl>
+        </p>
+      </desc>
+    </datatype>
+  </datatypes>
```

Comment:
For each time the `<datatypes> ...</datatypes>` block occurs a subtitle "Data Types" appears in the html doc. This is not what we want. It is better to have only one `<datatypes> </datatypes>` block and inside that using `<datatype_title>` several times like this:
```xml
<datatypes>
  <datatype_title>My first group of datatypes</datatype_title>
  <datatype>... </datatype>
  <datatype>... </datatype>
  ...
  <datatype_title>My second group of datatypes</datatype_title>
  <datatype>... </datatype>
  <datatype>... </datatype>
  ...
</datatypes>
```